topics/metabolomics/slides/introduction.html

```diff
@@ -0,0 +1,56 @@
+---
+layout: introduction_slides
+logo: "GTN"
+title: "Introduction to Metabolomics"
+type: "introduction"
+contributors:
+  - lecorguille
+---
+
+# What is the topic?
+
+---
+
+### Topic
+
+Blabla
+
+- Blabla
+- Blabla
+
+Blabla
+
+- Blabla
+- Blabla
```

Comment:
@lecorguille do you plan to fill these topic introduction slides as well? The slides are the only thing that will be visible in the topic until the tools for your tutorial are ready for the MTS, so some content in here would be nice. It doesn't have to be extensive, just couple slides about what metabolomics is would already be very helpful :)

Reply:
Done or so